### PR TITLE
Remove approvals requirement from mergable

### PR DESCRIPTION
**Pull request by ericholscher**
This just causes every PR to fail, which is terrible UX.

### CODE REVIEW

**Comment by stsewd** [comment]
I think there is a setting inside github to force approvals before merge

**Comment by stsewd** [comment]
There it is

screenshot_2019-01-30 stsewd dotfiles

**Comment by ericholscher** [comment]
Set it to require one review 👍

File: .github/mergeable.yml [hunk 1]
```diff
@@ -19,10 +19,11 @@ mergeable:
           # Do not allow empty descriptions on PR.
           enabled: true
           message: 'Description can not be empty.'
-      - do: approvals
-        min:
-          count: 1
-          message: 'The PR must have a minimum of 1 approvals.'
+# Removed because it causes every PR to fail
```
Comment (stsewd) [inline .github/mergeable.yml]: I guess we can just remove this lines since we have set up this from github

**Comment by humitos** [comment]
For labels, I think we should add in `must_exclude` the label `Status: blocked` also.

**Pull request merged**